db/objects/R__020_household_intake.sql

```diff
@@ -1,102 +1,110 @@
+        , sum(Addedsugar_in_g                / 100 * amount_consumed_in_g) as Addedsugar_in_g
+        , sum(Totalfibre_in_g                / 100 * amount_consumed_in_g) as Totalfibre_in_g
+        , sum(Starch_in_g                    / 100 * amount_consumed_in_g) as Starch_in_g
+        , sum(Ash_in_g                       / 100 * amount_consumed_in_g) as Ash_in_g
+        , sum(Ca_in_mg                       / 100 * amount_consumed_in_g) as Ca_in_mg
+        , sum(Fe_in_mg                       / 100 * amount_consumed_in_g) as Fe_in_mg
+        , sum(Mg_in_mg                       / 100 * amount_consumed_in_g) as Mg_in_mg
+        , sum(P_in_mg                        / 100 * amount_consumed_in_g) as P_in_mg
+        , sum(K_in_mg                        / 100 * amount_consumed_in_g) as K_in_mg
+        , sum(Na_in_mg                       / 100 * amount_consumed_in_g) as Na_in_mg
+        , sum(Zn_in_mg                       / 100 * amount_consumed_in_g) as Zn_in_mg
+        , sum(Cu_in_mg                       / 100 * amount_consumed_in_g) as Cu_in_mg
+        , sum(Mn_in_mcg                      / 100 * amount_consumed_in_g) as Mn_in_mcg
+        , sum(I_in_mcg                       / 100 * amount_consumed_in_g) as I_in_mcg
+        , sum(Se_in_mcg                      / 100 * amount_consumed_in_g) as Se_in_mcg
+        , sum(VitaminA_in_RAE_in_mcg         / 100 * amount_consumed_in_g) as VitaminA_in_RAE_in_mcg
+        , sum(VitaminA_in_RE_in_mcg          / 100 * amount_consumed_in_g) as VitaminA_in_RE_in_mcg
+        , sum(Thiamin_in_mg                  / 100 * amount_consumed_in_g) as Thiamin_in_mg
+        , sum(Riboflavin_in_mg               / 100 * amount_consumed_in_g) as Riboflavin_in_mg
+        , sum(Niacin_in_mg                   / 100 * amount_consumed_in_g) as Niacin_in_mg
+        , sum(VitaminB6_in_mg                / 100 * amount_consumed_in_g) as VitaminB6_in_mg
+        , sum(Folicacid_in_mcg               / 100 * amount_consumed_in_g) as Folicacid_in_mcg
+        , sum(VitaminB12_in_mcg              / 100 * amount_consumed_in_g) as VitaminB12_in_mcg
+        , sum(Pantothenate_in_mg             / 100 * amount_consumed_in_g) as Pantothenate_in_mg
+        , sum(Biotin_in_mcg                  / 100 * amount_consumed_in_g) as Biotin_in_mcg
+        , sum(VitaminC_in_mg                 / 100 * amount_consumed_in_g) as VitaminC_in_mg
+        , sum(VitaminD_in_mcg                / 100 * amount_consumed_in_g) as VitaminD_in_mcg
+        , sum(VitaminE_in_mg                 / 100 * amount_consumed_in_g) as VitaminE_in_mg
+        , sum(PhyticAcid_in_mg               / 100 * amount_consumed_in_g) as PhyticAcid_in_mg
+    FROM
+        fooditem
+        JOIN HOUSEHOLD_CONSUMPTION as hhc ON hhc.fooditem_id = fooditem.id
+        JOIN HOUSEHOLD_MEMBER as hhm ON hhm.id = hhc.HOUSEHOLD_id
+        JOIN household ON hhm.household_id = household.id
+        JOIN survey on household.survey_id = survey.id
```

Comment:
If the link between households and subregions is made less explicit (see comments below), this can then become something along the lines of: 

```suggestion
        JOIN survey on household.survey_id = survey.id
        JOIN subregion on ST_WITHIN(household.location, subregion.geometry)
```

allowing scope for future views/queries to use different aggregation areas